.pylintrc

```diff
@@ -0,0 +1,38 @@
+[MESSAGES CONTROL]
+
+disable=raw-checker-failed,
+        locally-disabled,
+        suppressed-message,
+        useless-suppression,
+        no-absolute-import,
+        unnecessary-pass,
+        invalid-name,
+        no-else-return,
+        missing-docstring,
+        no-self-use,
+        too-few-public-methods,
+        duplicate-code
+
+
+[FORMAT]
+
+max-line-length=99
+
+
+[BASIC]
+
+# ignore very short property getters and setters
+docstring-min-length=1
+
+
+[TYPECHECK]
+
+ignored-modules=pyanaconda
```

Comment:
This is the addon...

Reply:
```
# List of module names for which member attributes should not be checked
# (useful for modules/projects where namespaces are manipulated during runtime
# and thus existing member attributes cannot be deduced by static analysis. It
# supports qualified module names, as well as Unix pattern matching.
ignored-modules=
```

I don't think that pyanaconda qualifies as such a module. I am afraid that pylint will not be able to detect errors caused by invalid use of the pyanaconda code because of this.

Reply:
Removed, replaced by a note in the Makefile how to make it run if it fails